howso/direct/core.py

```diff
@@ -1649,6 +1650,15 @@ def batch_react_series(
             maximum size a series is allowed to be.  Default is
             3 * model_size, a 0 or less is no limit. Must provide
             either one for all series, or exactly one per series.
+        continue_series : bool, default False
+            When true will attempt to continue existing series instead of
+            starting new series. If initial_values provide series IDs, it
```

Comment:
`initial_values`

Reply:
er should be initial_values sorrounded by ``, if I type that into github comments it just highlights it